lib/storage-revocations.js

```diff
@@ -127,8 +146,18 @@ exports.isRevoked = async ({capabilities}) => {
     });
   }
 
-  const revoked = await _getCollection().findOne(
-    query, {projection: {_id: 1}});
+  const collection = _getCollection();
+  const projection = {_id: 1};
```

Comment:
I wrote this but I'm just now realizing that the query might not be covered. Which also makes me wonder if we can add anything to these tests to ensure that queries are covered (I'm pretty sure we've done that in ledger code before). So this, in particular, should be a covered query. Let's change this to be the capability ID:

```suggestion
  const projection = {'capability.id': 1};
```